lib/resources/r4/related_person.yaml

```diff
@@ -61,9 +61,6 @@ fields:
     - display: Millennium Person Relationship Types
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/351
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-351-person-relationship-types
-    - display: Millennium Person Relationships
-      system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/40
-      info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-40-person-relationships
```

Comment:
This is still supported.

Reply:
added back here https://github.com/cerner/fhir.cerner.com/pull/624/commits/906c8894c70e01bd63fcd1eae7377f2bd18849c5#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6R64-R66